spark/spark-spa/src/main/java/com/thoughtworks/go/spark/spa/spring/SpaControllers.java

```diff
@@ -58,6 +58,7 @@ public SpaControllers(SPAAuthenticationHelper authenticationHelper, VelocityTemp
         sparkControllers.add(new AnalyticsDelegate(authenticationHelper, templateEngineFactory.create(AnalyticsDelegate.class, defaultTemplate), systemEnvironment, analyticsExtension, pipelineConfigService));
         sparkControllers.add(new DataSharingSettingsDelegate(authenticationHelper, templateEngineFactory.create(DataSharingSettingsDelegate.class, defaultTemplate)));
         sparkControllers.add(new ConfigReposDelegate(authenticationHelper, templateEngineFactory.create(ConfigReposDelegate.class, componentAware), featureToggleService));
+        sparkControllers.add(new KitchenSinkDelegate(templateEngineFactory.create(KitchenSinkDelegate.class, () -> COMPONENT_LAYOUT_PATH)));
```

Comment:
@ketan - How about doing this:
```
if(!systemEnvironment.isProductionMode()) {
    sparkControllers.add(new KitchenSinkDelegate(templateEngineFactory.create(KitchenSinkDelegate.class, () -> COMPONENT_LAYOUT_PATH)));
}
```

This will not load the page in production.

Reply:
Don't care. Honestly. If someone gets there in production, let them.